src/screens/HomePage/HomepageStructuredData.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { getMapImageUrl } from 'common/urls';
+import { Helmet } from 'react-helmet';
+
+const HomepageStructuredData: React.FC = () => {
+  const jsonData = JSON.stringify({
+    '@context': 'https://schema.org',
+    '@type': 'ImageObject',
+    url: getMapImageUrl(),
+    width: 1200,
+    height: 630,
+  });
+
+  return (
+    <Helmet>
+      <script type="application/ld+json" charSet="UTF-8">
```

Comment:
The `charset` attribute is deprecated https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#deprecated_attributes, I think we can remove it